File: empl/empl/studdata/src/main/java/com/gabriel/studms/model/Student.java
package com.gabriel.studms.model;
import lombok.Data;
import com.fasterxml.jackson.annotation.JsonFormat;
import java.util.Date;

@Data
public class Student{
	int id;
    private String firstName;
    private String lastName;
    private String studentNumber;
    private String email;
    private String department;
    
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+08:00")
    private Date created;
    
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+08:00")
    private Date lastUpdated;
}

File: EmployeeApp/EmployeeApp/app/src/main/java/com/gabriel/employeeapp/data/model/Student.kt
package com.gabriel.employeeapp.data.model

data class Student(
    val id: Long? = null,
    val firstName: String,
    val lastName: String,
    val email: String,
    val department: String,
    val studentNumber: String,
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val imageUrl: String? = null
)